app/assignments/Assignments.js

```diff
@@ -82,28 +64,18 @@ export default function Assignments({ accessToken }) {
         <LoadingSpinner />
       </CommonLayout>
     )
+  if (configInvitation.apiVersion !== 2)
+    return <ErrorDisplay message={'Old version of assignments is not supported'} />
```

Comment:
Please use this message: "This venue was hosted using a deprecated API version, and the assignments are not available through this page. Please use our contact form if you need to access the assignment data."